README.md

```diff
@@ -0,0 +1,20 @@
+# Babylon liquidity tool (BLT)
+## overview
+The Babylon liquidity tool is a tool to accept user liquidity, distribute it throughout the wider ecosystem. The vault reads allocation weights from the gauge. The gauge interface is designed in such a way that it supports a wide range of datasources to calculate the allocations. Potential sources of allocations range from
```

Comment:
The Babylon Liquidity Tools accepts user liquidity and distributes it throughout the wider ecosystem.

Reply:
I would move the part "The vault reads [...] calculate the allocations" to the the section below (section mocks which I would rename to Design or Implementation)

Reply:
Potential sources of allocations range from -> This includes